src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -1396,45 +1399,33 @@ void runtime::cpu::CPU_ExternalFunction::build()
                 auto it = enable_nodename_list.begin();
                 for (const auto& p : enables)
                 {
-                    std::vector<std::function<void(CPURuntimeContext*)>> ftrs;
-                    for (size_t j = 0; j < p.second; j++)
-                    {
-                        ftrs.push_back(*functor);
-                        std::advance(functor, 1);
-                    }
-
                     tbb::flow::continue_node<tbb::flow::continue_msg, tbb::flow::lightweight>*
                         flowgraph_node = new tbb::flow::continue_node<tbb::flow::continue_msg,
                                                                       tbb::flow::lightweight>(
-                            *(ctx->G), [&, ftrs](const tbb::flow::continue_msg& msg) {
+                            *(ctx->G), [&](const tbb::flow::continue_msg& msg) {
                                 if (p.first(ctx) || ctx->first_iteration)
                                 {
-                                    for (size_t j = 0; j < p.second; j++)
```

Comment:
Is the mapped value needed elsewhere? If not, it doesn't need to be calculated and stored